src/components/core/form/FormForm.js

```diff
@@ -203,6 +209,7 @@ const fields = ({ formSlug, kappSlug, datastore }) => ({ form }) =>
             )
           : [],
       initialValue: get(form, 'type'),
+      helpText: 'Used for organizing forms. Values in the dropdown are defined under Form Types.'
     },
```

Comment:
Slight change to the first sentence.

Used for organizing and displaying forms.